highfidelity/hifi
# Use key codes for entity camera keys

Fixes: https://highfidelity.manuscript.com/f/cases/edit/17125

Test Plan:
- Enter Interface in desktop mode
- Open Create app and select any entity
- Press F on keyboard to focus camera on selected entity
- While in entity camera, press and hold the following keys to verify their expected behavior:

A:  orbit left
D:  orbit right
W:  orbit forward
S:  orbit backward
E:  orbit up
C:  orbit down
Left arrow:  orbit left
Right arrow:  orbit right
Up arrow:  orbit forward
Down arrow:  orbit backward

- For each above key, press and hold the key and verify that once clicking away from the Interface window the key is considered released
